docs/samples/bookinfo.md

```diff
@@ -110,54 +56,70 @@ _Note: The following instructions assume that you have access to a kubernetes cl
-   ```
+```bash
+$ kubectl get services
+NAME                       CLUSTER-IP   EXTERNAL-IP   PORT(S)              AGE
+details                    10.0.0.31    <none>        9080/TCP             6m
+istio-ingress-controller   10.0.0.122   <pending>     80:32000/TCP         8m
+istio-manager              10.0.0.189   <none>        8080/TCP             8m
+istio-mixer                10.0.0.132   <none>        9091/TCP,42422/TCP   8m
+kubernetes                 10.0.0.1     <none>        443/TCP              14d
+productpage                10.0.0.120   <none>        9080/TCP             6m
+ratings                    10.0.0.15    <none>        9080/TCP             6m
+reviews                    10.0.0.170   <none>        9080/TCP             6m
+```
 
    and
 
-   ```bash
-   $ kubectl get pods
-   NAME                                        READY     STATUS    RESTARTS   AGE
-   details-v1-1520924117-48z17                 2/2       Running   0          6m
-   istio-ingress-controller-3181829929-xrrk5   1/1       Running   0          8m
-   istio-manager-175173354-d6jm7               2/2       Running   0          8m
-   istio-mixer-3883863574-jt09j                2/2       Running   0          8m
-   productpage-v1-560495357-jk1lz              2/2       Running   0          6m
-   ratings-v1-734492171-rnr5l                  2/2       Running   0          6m
-   reviews-v1-874083890-f0qf0                  2/2       Running   0          6m
-   reviews-v2-1343845940-b34q5                 2/2       Running   0          6m
-   reviews-v3-1813607990-8ch52                 2/2       Running   0          6m
-   ```
+```bash
+$ kubectl get pods
+NAME                                        READY     STATUS    RESTARTS   AGE
+details-v1-1520924117-48z17                 2/2       Running   0          6m
+istio-ingress-controller-3181829929-xrrk5   1/1       Running   0          8m
+istio-manager-175173354-d6jm7               2/2       Running   0          8m
+istio-mixer-3883863574-jt09j                2/2       Running   0          8m
+productpage-v1-560495357-jk1lz              2/2       Running   0          6m
+ratings-v1-734492171-rnr5l                  2/2       Running   0          6m
+reviews-v1-874083890-f0qf0                  2/2       Running   0          6m
+reviews-v2-1343845940-b34q5                 2/2       Running   0          6m
+reviews-v3-1813607990-8ch52                 2/2       Running   0          6m
+```
 
 1. Determine the Gateway ingress URL
 
-   ```bash
-   $ export GATEWAY_URL=$(kubectl get po -l infra=istio-ingress-controller -o jsonpath={.items[0].status.hostIP}):$(kubectl get svc istio-ingress-controller -o jsonpath={.spec.ports[0].nodePort})
-   $ echo $GATEWAY_URL
-   192.168.99.100:32567
-   ```
+```bash
+export GATEWAY_URL=$(kubectl get po -l infra=istio-ingress-controller -o jsonpath={.items[0].status.hostIP}):$(kubectl get svc istio-ingress-controller -o jsonpath={.spec.ports[0].nodePort})
+```
```

Comment:
Explicit line continuations might make this easier to read on narrow screens, e.g.

```bash
export GATEWAY_URL=$(kubectl get pod -l infra=istio-ingress-controller \
         -o jsonpath={.items[0].status.hostIP}):$(kubectl get service istio-ingress-controller \
         -o jsonpath={.spec.ports[0].nodePort})
```